content/agent/basic_agent_usage/osx.md

```diff
@@ -119,14 +119,17 @@ Stop and Close the Datadog Agent: via the bone icon in the Tray.
 
 Drag the Datadog Application from the application folder to the Trash Bin.
 
-* $ sudo rm -rf /opt/datadog-agent
-* $ sudo rm -rf /usr/local/bin/datadog-agent
-* $ sudo rm -rf ~/.datadog-agent/**​ #to remove broken symlinks
-
-If you ran the optional install commands to have the Agent run at boot time, you also need to run the following to finish uninstalling:
+```
+$ sudo rm -rf /opt/datadog-agent
+$ sudo rm -rf /usr/local/bin/datadog-agent
+$ sudo rm -rf ~/.datadog-agent/**​ #to remove broken symlinks
+```
+If you ran the optional install commands to have the Agent run at boot time, run the following to finish uninstalling:
 
-* $ sudo launchctl unload -w /Library/LaunchDaemons/com.datadoghq.agent.plist
-* $ sudo  rm /Library/LaunchDaemons/com.datadoghq.agent.plist
+```
+$ sudo launchctl unload -w /Library/LaunchDaemons/com.datadoghq.agent.plist
+$ sudo  rm /Library/LaunchDaemons/com.datadoghq.agent.plist
```

Comment:
There's an extra space between `sudo` and `rm`: 

I would change to:

> $ sudo rm /Library/LaunchDaemons/com.datadoghq.agent.plist

Reply:
👌